CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Bugfix: Connection to Twitch PubSub now recovers more reliably. (#3643, #3716)
 - Minor: Added chatter count for each category in viewer list. (#3683, #3719)
 - Minor: Sorted usernames in /vips message to be case-insensitive. (#3696)
+- Minor: Strip leading @ from usernames in the `/block` and `/unblock` commands. (#3816)
```

Comment:
```suggestion
- Minor: Strip leading @ and trailing , from usernames in the `/block` and `/unblock` commands. (#3816)
```
Just gonna slap this suggestion in from 2.3.4 changelogs, even tho the trailing comma happens less often, the function still strips it